package.json

```diff
@@ -44,7 +44,8 @@
     "test:ts": "yarn jest --reporters=default --reporters=jest-junit",
     "test:iphone": "xcodebuild test -project swift/looker/looker.xcodeproj -scheme looker-Package -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 11,OS=13.4.1' | xcpretty --test --color",
     "test:jest": "DOT_ENV_FILE=.env.test jest",
-    "bootstrap": "lerna clean -y && lerna bootstrap"
+    "bootstrap": "lerna clean -y && lerna bootstrap",
+    "watch": "lerna run --parallel watch"
```

Comment:
You might be able to remove the watch scripts from all of the sub-projects by doing something like:

```
    "watch": "yarn lerna exec --scope '@looker/*' --stream 'BABEL_ENV=build babel src --root-mode upward --out-dir lib --source-maps --extensions .ts,.tsx --no-comments' --watch",
```

Reply:
That would be handy

Reply:
Unfortunately that throws saying that `watch` is an unknown lerna argument

Reply:
and if `--watch` is moved to be part of the babel cmd then it is left hanging on the second package build:

```
$ yarn lerna exec --scope '@looker/*' --stream 'BABEL_ENV=build babel src --root-mode upward --out-dir lib --source-maps --extensions .ts,.tsx --no-comments --watch'
lerna notice cli v3.22.1
lerna info versioning independent
lerna notice filter including "@looker/*"
lerna info filter [ '@looker/*' ]
lerna info Executing command in 7 packages: "BABEL_ENV=build babel src --root-mode upward --out-dir lib --source-maps --extensions .ts,.tsx --no-comments --watch"
@looker/sdk-codegen-utils: Successfully compiled 2 files with Babel (545ms).
@looker/sdk: Successfully compiled 34 files with Babel (3907ms).
```